generic-skiplink/skiplink.css

```diff
@@ -1,18 +1,32 @@
 a[skiplink] {
-  background-color: white;
-  border-radius: 5px;
-  padding: 10px 20px 10px 20px;
-  position: absolute;
-  left: -999px;
-  width: 1px;
-  height: 1px;
-  top: auto;
+  position: fixed !important;
+  /* keep it on viewport */
+  top: 0px !important;
+  left: 0px !important;
+  /* give it non-zero size, VoiceOver on Safari requires at least 2 pixels
+     before allowing buttons to be activated. */
+  width: 4px !important;
+  height: 4px !important;
+  /* visually hide it with overflow and opacity */
+  opacity: 0 !important;
+  overflow: hidden !important;
+  /* remove any margin or padding */
+  border: none !important;
+  margin: 0 !important;
+  padding: 0 !important;
+  /* ensure no other style sets display to none */
+  display: block !important;
+  visibility: visible !important;
+  pointer-events: none !important;
 }
 
 a[skiplink]:focus {
```

Comment:
Is the intention to try to avoid opinionated styles in the default CSS? In that case maybe this shouldn't set a `background-color`? Of course that could make the text hard to read without further customization due to `position: absolute` (which I think appropriately overrides `fixed`), as opposed to the otherwise default `static`.

Reply:
Should `display` perhaps be (re)set to the UA default of `inline` in the `:focus` state? Currently `block` is inherited from the declaration in the "hidden" state.

Reply:
Set `pointer-events: auto` here, although `:focus` probably implies keyboard navigation, it doesn't stop the user from using a mouse pointer on the link when it's presented to them.

Reply:
- RE: `background-color`:
I think _some_ opinionated style for the skiplink should be fine, so long as its easily overrideable. I probably should remove the `!important` from there, so that the user can just do `a[skiplink]{background-color: red;}`

- RE: `pointer-events: auto`:
Do you mean set `pointer-events: auto` in the `:focus` style?

Reply:
> set pointer-events: auto in the :focus style?

Yes.

Reply:
> - RE: `background-color`:
> I think some opinionated style for the skiplink should be fine

Do you need to consider dark mode here?

Reply:
Yep, makes sense. Gotcha. These are fixed in https://github.com/thepassle/generic-components/pull/35/commits/627cbd202b0b6bc73429a628fd833e71a23c7aae 🙂